.github/workflows/ci.yml

```diff
@@ -6,8 +6,13 @@ jobs:
     name: PHP ${{ matrix.php }} - doctrine/orm ${{ matrix.orm }}
     strategy:
       matrix:
-        php: ['7.2', '7.3', '7.4']
-        orm: ['2.6.0', '2.7.0']
+        php: ['7.2', '7.3', '7.4', '8.0']
+        orm: ['2.6.0', '2.7.0', '2.8.0']
+      exclude:
+        - php: '8.0'
+          orm: '2.6.0'
+        - php: '8.0'
+          orm: '2.7.0'
```

Comment:
Should be with same indention as "orm". Take a look here:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix

Reply:
Ahhh, thankyou. See how the next build goes.

Reply:
Try to allow phpunit 8 for PHP 7.2 support.

Reply:
Whoops, was testing the composer require step locally and committed those changes.